spinalcordtoolbox/reports/qc.py

```diff
@@ -558,7 +558,7 @@ def __init__(self, input_file, command, args, orientation, dest_folder, dpi=300,
         self.dpi = dpi
         self.root_folder = dest_folder
         self.mod_date = datetime.datetime.strftime(datetime.datetime.now(), '%Y_%m_%d_%H%M%S.%f')
-        self.qc_results = os.path.join(dest_folder, '_json/qc_' + self.mod_date + '.json')
+        self.qc_results = os.path.join(dest_folder, '_json', 'qc_' + self.mod_date + '.json')
```

Comment:
A tiny nitpick: the filename part of the path looks like a good candidate for an f-string.
```suggestion
        self.qc_results = os.path.join(dest_folder, '_json', f'qc_{self.mod_date}.json')
```